helm/amster/values.yaml

```diff
@@ -31,10 +27,12 @@ config:
   name: frconfig
   # The path where we import the configuration from. Your git repo will usually be mounted on /git/config.
   importPath: /git/config/6.5/default/am/empty-import
-  # Where we perform export paths to. If not set, this defaults to the importPath
+  # Where we perform export to. If not set, this defaults to the importPath
```

Comment:
Sugg: s/Where we perform export to/The path to which configuration is exported./